packages/terminology/src/sqlite.ts

```diff
@@ -0,0 +1,64 @@
+import { join } from "path";
+import { mkdirSync } from "fs";
+
+import { SqliteClient } from "./sqlClient";
+import * as migrations from "./migrations/index";
+import { seedCodeSystems } from "./seed";
+
+let dbClient: SqliteClient | undefined;
+
+type Migration = {
+  sql: string;
+};
+
+async function migrate(dbClient: SqliteClient): Promise<void> {
```

Comment:
migration table stores current version. 
If version file v5.ts exists, but we are only on v4, then we run the sql query in v5.ts